ui/pages/notifications/notifications.js

```diff
@@ -130,13 +130,13 @@ export default function Notifications() {
 }
 
 NotificationItem.propTypes = {
-  notification: {
+  notification: PropTypes.exact({
```

Comment:
Is this change required for the work of making the design-system enums?

Reply:
reverted 🙏